packages/data-transport-layer/src/utils/eth-tx.ts

```diff
@@ -0,0 +1,9 @@
+/* Imports: External */
+import { ethers } from 'ethers'
+
+export const parseSignatureVParam = (
+  v: number | ethers.BigNumber,
+  chainId: number
+): number => {
+  return ethers.BigNumber.from(v).toNumber() - 2 * chainId - 35
+}
```

Comment:
I think this could use a docstring to explain what "parse" really means. i.e., why is `v` not already "parsed" before it was provided to this function? For a newcomer to the codebase it's rather confusing how this function ended up being needed.